Appraisals

```diff
@@ -9,3 +9,7 @@ end
 appraise 'rails-7.1' do
   gem 'rails', '~> 7.1.0'
 end
+
+appraise 'rails-7.2' do
+  gem 'rails', github: 'rails/rails'
```

Comment:
Actually right now `rails/rails` points to 8.0.0.alpha whereas `7-2-0-stable` points to `7.2`